src/SFA.DAS.AdminService.Web/Startup.cs

```diff
@@ -109,24 +113,18 @@ public void ConfigureServices(IServiceCollection services)
                     o.ViewLocationFormats.Add("/Views/Application/{0}" + RazorViewEngine.ViewExtension);
                 });
 
-            services.AddSession(opt => 
+            services.AddSession(opt =>
             {
                 opt.IdleTimeout = TimeSpan.FromHours(1);
             });
 
-            if (!_env.IsDevelopment())
```

Comment:
why is this logic removed

Reply:
_env is passed down and the check is done inside the extension to determine whether or not to use redis for both session caching and data protection keys.